starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,32 @@
+/* reset */
+
```

Comment:
Reset is a way to reduce browser inconsistencies. We will see more advanced stuff. For now, you can do:
```
/* ---- reset ---- */
h1, body, html, main {
  margin: 0;
}
```